app/src/components/BaseImageUpload.vue

```diff
@@ -46,7 +46,7 @@
 
       <!-- filename -->
       <div class="col-span-6 md:col-span-4 mb-3 md:mb-0 truncate">
-        {{ logoURI }}
+        {{ logoURI !== '/placeholder_grant.svg' ? logoURI : 'placeholder_grant.svg' }}
```

Comment:
should the falsy branch be prefixed with a `/`?

Reply:
I'm not sure if it needs to be, it was working find without `/` but was worried it could break in prod - what are your thoughts?

Reply:
Ahhh this is only used to display the filename - yeah we don't need it (the `/`)!